### PR TITLE
[sdk/python]: partially lint generated code

### PR DESCRIPTION
**Pull request by Jaguar0625**

### CODE REVIEW

File: sdk/python/generator/Generator.py [hunk 1]
```diff
@@ -39,6 +39,10 @@ def generate_files(ast_models, output_directory: Path):
 			'''#!/usr/bin/python
 #
 # Code generated by catbuffer python generator; DO NOT EDIT.
+#
+# pylint: disable=line-too-long, invalid-name, redefined-builtin
```
Comment (gimre-xymcity) [inline sdk/python/generator/Generator.py:43]: hmm, where does it complain about redefiend builtin?

File: sdk/python/generator/FactoryFormatter.py [hunk 2]
```diff
@@ -86,7 +86,7 @@ def get_create_by_name_descriptor(self):
 		body += indent(
 			',\n'.join(
 				map(
-					lambda child: f'"{skip_embedded(underline_name(child.name))}": {child.name}',
+					lambda child: f'\'{skip_embedded(underline_name(child.name))}\': {child.name}',
```
Comment (gimre-xymcity) [inline sdk/python/generator/FactoryFormatter.py:89]: hate escaped strings 😬

**Review by gimre-xymcity — APPROVED** [review]

File: sdk/python/generator/Generator.py (hunk 1 above)
Comment (Jaguar0625) [reply sdk/python/generator/Generator.py:43]: hash

File: sdk/python/generator/FactoryFormatter.py (hunk 2 above)
Comment (Jaguar0625) [reply sdk/python/generator/FactoryFormatter.py:89]: it makes the output look prettier 😅

**Pull request merged**